packages/react/src/components/ComboBox/ComboBox.js

```diff
@@ -191,7 +191,7 @@ export default class ComboBox extends React.Component {
      * Provide text to be used in a `<label>` element that is tied to the
      * combobox via ARIA attributes.
      */
-    titleText: PropTypes.string,
+    titleText: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
```

Comment:
I believe a string will still be considered a `node` and not throw any warnings

```suggestion
    titleText: PropTypes.node,
```

Reply:
it was lifted directly from the dropdown but I updated them all across the board